pkg/util/display/bytes.go

```diff
@@ -48,3 +56,13 @@ func BytesToString(b uint64) string {
 func formatStorageString(b, u uint64, unitLabel string) string {
 	return fmt.Sprintf("%.2f %s", float64(b)/float64(u), unitLabel)
 }
+
+// BToMb converts bytes to megabytes
+func BToMb(b uint64) uint64 {
+	return b / 1024 / 1024
```

Comment:
I wonder if there are division issues possible due to not doing floating division. Would it be safer to do something like below?
```go
mb := float(b) / 1024.0 / 1024.0
return uint64(math.Round(mb))
```

Reply:
To be honest, I don't know :)

Reply:
Although I don't realistically expect too much of an issue, I think it would be safest to do the rounding just in case. It also occurs to me that the round may be down to zero; do we want to return a minimum of 1 Mb?

Reply:
For the purposes of the memory limits, seems sufficient to keep it simple with what's already implemented. There is a check elsewhere for the minimum. But if this helper is used more generally, it would certainly be more accurate to round as suggested.